content/learn.md

```diff
@@ -44,6 +46,7 @@ DataCamp is an excellent way to improve your R skills, including the tidyverse.
 ## Workshops
 
   * [Mastering the Tidyverse](https://www.jumpingrivers.com/courses/22_r-tidyverse) by Jumping Rivers. A one day crash course covering tidyverse fundamentals. The course is a mixture of lectures, short exercises and longer tutorial questions. During the day, we'll cover dplyr, tidy data, tibbles, dates/times and string manipulation.
+  * [Introduction to R](https://itsalocke.com/courses/intro-to-r/) by Locke Data. A two day course covering data manipulation and reporting fundamentals using the tidyverse, rmarkdown and shiny. The course blends lectures, exercises, and practicals over two days to cover the 80% of work that almost everyone needs to do.
```

Comment:
🤓 Tidyverse copy-editor here, can I just request an oxford comma in `rmarkdown, and shiny`? It's house style, and now that I see the one above is missing it, I'll change that, too!

Reply:
I didn't oxford comma because the prior one didn't. I'm *very happy* to add it in as it's the RIGHT WAY :)